### PR TITLE
Simulate dirt neutrinos in MPV

### PR DESCRIPTION
**Pull request by bear-is-asleep**
## Description 
Simulate MPV upstream of detector to train for dirt neutrinos.

Screenshot 2025-04-14 at 3 50 41 PM
^^ shows reconstructed neutrino vertex from Pandora after selection. Most are within 50 cm of TPC face in z-direction. 
image

^^ shows the density of neutrino interactions by true type after fv cut in pandora selection.
See docdb [36641](https://sbn-docdb.fnal.gov/cgi-bin/sso/ShowDocument?docid=36641)

This expands the volume by 75% `(440*440*540)/(360*360*460)=1.753`, so one would need to scale the training dataset size by 75% more events to reproduce a similar training set of the old volume (for the rain/MPR sample).

## Checklist
- [ ] Added at least 1 label from [available labels](https://github.com/SBNSoftware/sbndcode/issues/labels?sort=name-asc).
- [ ] Assigned at least 1 reviewer under `Reviewers`,
- [ ] Assigned all contributers including yourself under `Assignees`
- [ ] Linked any relevant issues under `Developement`
- [ ] Does this PR affect CAF data format? If so, please assign a CAF maintainer ([PetrilloAtWork](https://github.com/PetrilloAtWork) or [JosiePaton](https://github.com/JosiePaton)) as additional reviewer.
- [ ] Does this affect the standard workflow? 

### Link(s) to docdb describing changes (optional)
docdb [36641](https://sbn-docdb.fnal.gov/cgi-bin/sso/ShowDocument?docid=36641)

### CODE REVIEW

**Comment by bear-is-asleep** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v10_04_07 SBNSoftware/sbncode@v10_04_07 SBNSoftware/sbnalg@v10_04_07 SBNSoftware/sbnobj@v10_00_08

**Comment by bear-is-asleep** [comment]
Approved

**Comment by nathanielerowe** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v10_04_08 SBNSoftware/sbncode@v10_04_08 SBNSoftware/sbnalg@v10_04_08 SBNSoftware/sbnobj@v10_00_09

**Comment by nathanielerowe** [comment]
@mrmooney @francescopoppi @justinjmueller I was told that this was approved via slack, but haven't actually seen the thread. Can someone confirm that this is good to go?

**Comment by nathanielerowe** [comment]
trigger build LArSoft/lar*@LARSOFT_SUITE_v10_05_00 SBNSoftware/sbncode#533

**Comment by francescopoppi** [comment]
> @mrmooney @francescopoppi @justinjmueller I was told that this was approved via slack, but haven't actually seen the thread. Can someone confirm that this is good to go?

Hi, are you sure it was me? I think there was a typo, since this is all SBND stuff

**Comment by nathanielerowe** [comment]
@francescopoppi Yes, sorry wrong person! My mistake!

**Comment by nathanielerowe** [comment]
Merging after talking with Bear about approval
